.env

```diff
@@ -0,0 +1,3 @@
+ifndef MODULE_VERSION
```

Comment:
Isn't it confusing that we have defaults defined in the Makefile itself and the .env file?
What's the benefit for us?

Reply:
You're right, it might be a little confusing, we can adapt that if we want to